From bc5adee4c17eafaca1c8ce1a6bd89143b8c3d44f Mon Sep 17 00:00:00 2001
From: Todd <toddrme2178@gmail.com>
Date: Mon, 2 Dec 2019 12:05:32 -0500
Subject: [PATCH] Use current python executable

Currently the system default is used, which may not be the version running the test and may not have pypipegraph installed.
---
 tests/test_other.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/test_other.py b/tests/test_other.py
index 2ab70df..b6ea8d5 100644
--- a/tests/test_other.py
+++ b/tests/test_other.py
@@ -24,6 +24,7 @@
 
 import os
 import subprocess
+import sys
 import pytest
 import pypipegraph as ppg
 from .shared import write, read, append, Dummy, assertRaises
@@ -189,7 +190,7 @@ def testing_import_does_not_hang(self):  # see python issue22853
         old_dir = os.getcwd()
         os.chdir(os.path.dirname(__file__))
         p = subprocess.Popen(
-            ["python", "_import_does_not_hang.py"],
+            [sys.executable, "_import_does_not_hang.py"],
             stdout=subprocess.PIPE,
             stderr=subprocess.PIPE,
         )
